mypy/fastparse.py

```diff
@@ -647,13 +652,25 @@ def visit_Num(self, n: ast35.Num) -> Node:
     # Str(string s) -- need to specify raw, unicode, etc?
     @with_line
     def visit_Str(self, n: ast35.Str) -> Node:
-        return StrExpr(n.s)
+        if self.pyversion[0] >= 3 or self.is_stub:
+            # Hack: assume all string literals in Python 2 stubs are normal
+            # strs (i.e. not unicode).  All stubs are parsed with the Python 3
+            # parser, which causes unprefixed string literals to be interpreted
+            # as unicode instead of bytes.  This hack is generally okay,
+            # because mypy considers str literals to be compatible with
+            # unicode.
+            return StrExpr(n.s)
+        else:
+            return UnicodeExpr(n.s)
 
     # Bytes(bytes s)
     @with_line
     def visit_Bytes(self, n: ast35.Bytes) -> Node:
         # TODO: this is kind of hacky
```

Comment:
The "this is hacky" comment is in kind of a weird place now.  It refers to the slicing that happens in the following lines.

That comment cleanup sounds nice.  I'll add that to the PR.